### DISCUSSION TITLE
Welcome to Garden_Defender Discussions!

### DISCUSSION THREAD

**Discussion by Ernestoflores27** [Ernestoflores27/Garden_Defender#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by maydayVFR** [comment]
Hey guys! This is absolutely brilliant! It's such a well put together project. The documentation is thorough which is fantastic. It really deserves more exposure. I've reviewed it a few times and I'm all in! Hopefully I can finish it sometime soon! I'll let you know when I get stuck. Thank you!
